nilearn/input_data/base_masker.py

```diff
@@ -145,6 +147,12 @@ def transform_single_imgs(self, imgs, confounds=None, copy=True):
             documentation for details.
             shape: (number of scans, number of confounds)
 
+        sample_mask : Any type compatible with numpy-array indexing, optional
+            Masks the niimgs along time/fourth dimension to perform scrubbing
+            (remove volumes with high motion) and/or non-steady-state volumes.
+            This parameter is passed to signal.clean.
+            shape: (number of scans - number of volumes removed, )
```

Comment:
I think it is mroe common to put the shape before the explanation text in the docstring.